vaadin-grid.html

```diff
@@ -6,6 +6,19 @@
 
 <link rel='import' href='../polymer/polymer.html'>
 <script src="vaadin-grid.min.js"></script>
+<script>
+if (!window.vaadin) {
```

Comment:
This is an issue using the component in vaadin framework, since window.vaadin already exists